privacy-protections/surrogates/main.js

```diff
@@ -55,6 +55,15 @@ const surrogates = {
         test: () => { return !!(window.ga && Object.keys(window.ga.create()).length === 0); },
         cleanUp: () => { delete window.ga; }
     },
+    'google-analytics.com/analytics.js broken integrity': {
+        url: 'https://google-analytics.com/analytics.js',
+        crossOrigin: 'anonymous',
+        integrity: 'sha512-1xNTXD/ZeaKg/Xjb6De9la7CXo5gC1lMk+beyKo691KJrjlj0HbZG6frzK0Wo6bm96i9Cp6w/WB4vSN/8zDBLQ==',
+        notes: 'Fails loading in all browsers despite we think it should be possible for the extension to load this.',
+        shouldFail: false,
+        test: () => { return !!(window.ga && Object.keys(window.ga.create()).length === 0); },
```

Comment:
This will be green if surrogate successfully loads, right? It will still be red if extension is disabled and original loads?

Reply:
Yes, although the integrity isn't valid also so shouldn't pass either.

Reply:
AKA this should only pass if the extension redirecting to a local extension resource is carved out within the browser code